tests/unit/multi_select_test.coffee

```diff
@@ -1,36 +1,34 @@
-multiselect = null
-dispatcher = null
-
-module "[Unit] Multi-select unit tests",
-  setup: ->
-    Ember.run ->
-      dispatcher = Ember.EventDispatcher.create()
-      dispatcher.setup()
-      multiselect = Ember.Widgets.MultiSelectComponent.create
-        content: ['foo', 'bar', 'barca', 'baz']
-
-  teardown: ->
-    Ember.run ->
-      dispatcher.destroy()
-      multiselect.destroy()
+moduleForComponent 'multi-select', '[Unit] Multi select component'
 
 test 'Test preparedContent after some options are already selected', ->
   expect 2
+  multiselect = @subject
```

Comment:
We used to not need to create the component for every single test. As I understand, @subject is an alias of create the component. If it is, is it good?